jenkins/branches/demo/1.env

```diff
@@ -0,0 +1,2 @@
+FXA_IFRAME_SRC="https://stomlinson.dev.lcip.org/"
```

Comment:
I think you need to omit the double quotes here: `FXA_IFRAME_SRC=https://stomlinson.dev.lcip.org/`

Reply:
Removed. I had the double quotes in my own .env file when testing against stomlinson.dev.lcip.org, so kept them. Wasn't sure if that was necessary or not.